packages/x64dbg.dbgchild.vm/tools/chocolateyuninstall.ps1

```diff
@@ -0,0 +1,15 @@
+$ErrorActionPreference = 'Continue'
+
+$toolDir = Join-Path ${Env:RAW_TOOLS_DIR} 'x64dbg\release'
+foreach ($file in $("NewProcessWatcher.exe", "readme_dbgchild.txt", "x64_post.unicode.txt", "x64_pre.unicode.txt", "x86_post.unicode.txt", "x86_pre.unicode.txt", "dbgchildlogs")) {
+    Remove-Item "${toolDir}\${file}" -Recurse -Force -vb
```

Comment:
Not sure what the `-vb` do and if they're necessary

Reply:
removed